base/commands/config/config_import.go

```diff
@@ -118,12 +118,14 @@ func (cm ImportCmd) tryImportViridianZipSource(ctx context.Context, ec plug.Exec
 }
 
 func (cm ImportCmd) download(ctx context.Context, ec plug.ExecContext, url string) (string, error) {
-	p, stop, err := ec.ExecuteBlocking(ctx, "Downloading the sample", func(ctx context.Context) (any, error) {
+	p, stop, err := ec.ExecuteBlocking(ctx, func(ctx context.Context, sp clc.Spinner) (any, error) {
+		sp.SetText("Creating temporary directory")
```

Comment:
Of course our hint logic is a little different, we provide info for basic operations too.  It might not feasible to produce this much output for basic `ExecuteBlocking` calls but we can add this as an option.

Reply:
Yes, this is one of the reasons to change the ExecuteBlocking signature to pass a Spinner.